warehouse/deltalake/deltalake.go

```diff
@@ -14,20 +14,20 @@ import (
 	"github.com/rudderlabs/rudder-server/utils/logger"
 	"github.com/rudderlabs/rudder-server/utils/misc"
 	"github.com/rudderlabs/rudder-server/warehouse/client"
-	"github.com/rudderlabs/rudder-server/warehouse/deltalake/databricks"
+	"github.com/rudderlabs/rudder-server/warehouse/deltalake/deltalakeclient"
 	warehouseutils "github.com/rudderlabs/rudder-server/warehouse/utils"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
-	"google.golang.org/grpc/health/grpc_health_v1"
 )
 
 // Database configuration
 const (
-	DLHost                 = "host"
```

Comment:
We can also move the deltalake package under and `integration` folder, to manage the clutter on top-level warehosue package

Reply:
I am thinking once this PR is merged, Create a separate PR for the change since there will to too many changes.

Reply:
https://github.com/rudderlabs/rudder-server/pull/2885